src/Identity/UI/src/IdentityBuilderUIExtensions.cs

```diff
@@ -131,5 +125,31 @@ private static void AddRelatedParts(IdentityBuilder builder, UIFramework framewo
                     }
                 });
         }
+
+        private static Assembly GetApplicationAssembly(IdentityBuilder builder)
+        {
+            // Whis is the same logic that MVC follows to find the application assembly.
```

Comment:
~~Whis~~ This

/cc @davidfowl since this is your favourite piece of code

Reply:
I'm getting rid of it

Reply:
couldn't get rid of it, but unimportant as we only poke the collection in testing scenarios. (otherwise app.getentryassembly works fine)